pkg/repositories/repositories_test.go

```diff
@@ -12,15 +12,19 @@ func newTestRepoObject(name string) Repository {
 			Name:        name + " Name",
 			Description: name + " Description",
 		},
-		URL:     name + " URL",
-		Version: name + " Version",
+		URL:        name + " URL",
+		Version:    name + " Version",
+		UpgradeURL: name + " Upgrade Url",
 	}
 }
 
 func testfileExpectedConfig() Config {
```

Comment:
Not a blocker: Could we make it a var instead of a function, since we use it in a static way? Might not be worth the loss in readability though.

Reply:
I think we've been leaning towards using functions for creation of objects that are reused mostly to prevent any changes to those object between the tests. I will take a look at this maybe again next time I'm in this section but feel free to do the same.